packages/utils/src/helpers.test.ts

```diff
@@ -420,3 +421,76 @@ describe('preformatWithRequestID function', () => {
     expect(preformatWithRequestID(undefined, expectedText)).toEqual(expectedText);
   });
 });
+
+describe('dateRangeToUnix function', () => {
+  const timestamps = {
+    JAN_1_2025_START: 1735689600, // 2025-01-01 00:00:00 UTC
+    JAN_31_2025_END: 1738367999, // 2025-01-31 23:59:59 UTC
+    JUN_15_2025_START: 1749945600, // 2025-06-15 00:00:00 UTC
+    JUN_16_2025_END: 1750118399 // 2025-06-16 23:59:59 UTC
+  };
+
+  const timezones = [
+    'America/New_York',
+    'Asia/Tokyo',
+    'Australia/Sydney',
+    'Europe/London',
+    'Asia/Dubai',
+    'Pacific/Auckland',
+    'Africa/Cairo',
+    'America/Los_Angeles',
+    'Asia/Shanghai',
+    'Europe/Paris',
+    'UTC'
+  ];
+
+  it('converts valid date strings to unix timestamps with different timezones', () => {
+    for (const timezone of timezones) {
+      process.env.TZ = timezone;
+      const result = dateRangeToUnix('2025-01-01', '2025-01-31');
+      expect(result.start).toBe(timestamps.JAN_1_2025_START);
+      expect(result.end).toBe(timestamps.JAN_31_2025_END);
+    }
+  });
+
+  it('handles Date objects', () => {
+    for (const timezone of timezones) {
+      process.env.TZ = timezone;
+      const startDate = new Date('2025-06-15');
```

Comment:
I've tried new Date('2025-03-06T00:50:00') for Oslo Timezone and recieved `.start` timestamp representing `Wednesday, 5 March 2025 00:00:00` :(

Reply:
good catch, may bad not including time in the tests